index.js

```diff
@@ -190,40 +182,37 @@ async function apostrophe(options, telemetry, rootSpan) {
       // are already recorded by the event module instrumentation
       console.error('beforeExit emit error', e);
     }
-    if (error) {
-      telemetry.aposHandleError(rootSpan, error);
-    } else if (code !== 0) {
-      rootSpan.setStatus({
-        code: telemetry.SpanStatusCode.ERROR
-      });
+
+    if (code !== 0) {
+      telemetry.handleError(rootSpan, message);
     } else {
       rootSpan.setStatus({
-        code: telemetry.SpanStatusCode.OK
+        code: telemetry.api.SpanStatusCode.OK,
+        message
       });
     }
-
-    // Exhaust the exit queue stack
-    let fn = self._onExitQueue.pop();
-    while (fn) {
-      try {
-        await fn();
-      } catch (e) {
-        console.error('_onExitQueue handler error', e);
-      }
-      fn = self._onExitQueue.pop();
-    }
     rootSpan.end();
 
     if (typeof options.beforeExit === 'function') {
       try {
-        await options.beforeExit(code, error);
+        await options.beforeExit(code, message);
       } catch (e) {
         console.error('beforeExit handler error', e);
       }
     }
     process.exit(code);
   };
 
+  process.once('SIGINT', (code) => {
```

Comment:
We should not set these handlers at least not at this time. There would be interactions with multisite and we are way past our scope already in covering the startup part of tracing so thoroughly for this week.